packages/core/src/connectors/index.ts

```diff
@@ -11,15 +11,15 @@ export const getConnectorById = (id: string): ConnectorInstance | null => {
 
 export const initConnectors = async () => {
   await Promise.all(
-    connectors.map(async ({ metadata: { id, type } }) => {
-      const record = await findConnectorByIdAndType(id, type);
+    connectors.map(async ({ metadata: { id, ...rest } }) => {
+      const record = await findConnectorById(id);
       if (record) {
         return;
       }
 
       await insertConnector({
         id,
-        type,
+        ...rest,
```

Comment:
seems no need to save `rest`? Only `type`.

Reply:
make sense